depends/packages/gmp.mk

```
@@ -1,13 +1,14 @@
 package=gmp
-$(package)_version=6.2.1
-$(package)_download_path=https://gmplib.org/download/gmp
+$(package)_version=6.3.0
+$(package)_download_path=https://ftp.gnu.org/gnu/gmp
 $(package)_file_name=gmp-$($(package)_version).tar.bz2
-$(package)_sha256_hash=eae9326beb4158c386e39a356818031bd28f3124cf915f8c5b1dc4c7a36b4d7c
+$(package)_sha256_hash=ac28211a7cfb609bae2e2c8d6058d66c8fe96434f740cf6fe2e47b000d1c20cb
 
 define $(package)_set_vars
-$(package)_config_opts+=--enable-cxx --enable-fat --disable-shared
-$(package)_cflags_armv7l_linux+=-march=armv7-a
-$(package)_cflags_aarch64_darwin+=-march=armv8-a
+$(package)_config_opts += --disable-shared --enable-cxx --enable-fat --with-pic
```

we merged https://github.com/dashpay/dash/pull/6293/commits/cd33b69bfdee7d52ea9c291e471ab6f7cd4bfc75 recently so `--with-pic` is not needed here
```suggestion
$(package)_config_opts += --disable-shared --enable-cxx --enable-fat
```

---

Resolved in latest push!